### ISSUE TITLE
Issue: Theme options in WPGraphQL with Carbon Fields

### ISSUE THREAD

**Issue by tysian** [matepaiva/wp-graphql-crb#1]
Hello!
I've decided to use your plugin along with Carbon fields (v. 3.2) and WPGraphQL. 
Unfortunately the `theme_options` field doesn't show up in the GraphQL IDE in my wordpress (v. 5.6).
My theme options container looks like this:

```php
WpGraphQLCrbContainer::register(
    Container::make( 'theme_options', __( 'Theme Options' ) )
        ->add_tab( 'Footer', [
            Field::make( 'text', 'footer_text', 'Footer text' ),
        ] )
);
```
Adding `->set_visible_in_rest_api(true)` didn't change anything.
Is this something expected, maybe you can help me with this problem?

Kind regards,
Jakub.

**Comment by matepaiva** [comment]
Hi @tysian!

Glad to hear that you are using it. I made this plugin one year ago and then I didn't touch it anymore. Basically, I was the only person using it ─ as far as I know ─ and I didn't need to expose the theme options through GraphQL.

So, the `theme_options` were not contemplated in that first version. I would add it later... and "later" never came, heh.

I will read the code again to try to understand what I would have to do to add support to theme options. I guess it's going to be easy.

Also, if you want to try adding the feature and sending a PR, you are most welcome.

**Comment by matepaiva** [comment]
Hello, @tysian. I implemented a solution that now exposes `theme_options` to GraphQL. Please update your dependencies. As I am not working with versioning yet, you might have some trouble updating the code. I will address it tomorrow.

Let me know if that is enough for your usage.

Also, are you installing the plugin via packagist? The readme had wrong information, but it is now up to date.

**Comment by tysian** [comment]
Hey @matepaiva!

Thank you very much for fast response! Theme Options were pretty important for me, because they were replacement for ACF Pro Options...
I've tried to fork your project and fix it by myself, but I got lost somewhere on `Container.php` haha :)
But after your update, I removed previous version from `vendor` directory and run `composer require matepaiva/wp-graphql-crb:dev-master` and everything works perfect! You rescued me from buying ACF Pro or using different CMS 👍

Thanks!
